app/src/test/java/com/vero/tasky/auth/data/repository/AuthRepositoryFake.kt

```diff
@@ -0,0 +1,20 @@
+package com.vero.tasky.auth.data.repository
+
+import com.vero.tasky.auth.domain.repository.AuthRepository
+import com.vero.tasky.core.domain.model.User
+
+class AuthRepositoryFake : AuthRepository {
```

Comment:
You should have a way to force the return to be Failure also

Because if you test viewmodels (and you send a FakeRepository to it) you need to verify it works for both Success and Failure